pkg/linter/lintFilterRuleSplitSubnet.go

```diff
@@ -84,7 +82,7 @@ func ruleSplitSubnet(subnet vpcmodel.Subnet, ruleIPBlocks []*ipblock.IPBlock) bo
 //////////////////////////////////////////////////////////
 
 func (finding *splitRuleSubnet) vpc() []string {
-	return []string{finding.vpcName}
+	return []string{finding.splitSubnets[0].VPC().Name()}
```

Comment:
1. rename function to vpcName()
2. why does this function return a slice? if there is a reason, rename it to vpcsNames()

Reply:
It returns a slice since in multi-vpc configs there could be more than one relevant vpc. 
The interface was agreed upon with @zivnevo; however, reading your comment and revisiting this function:
1. Wouldn't it indeed make more sense to return the VPC and not its name?
2. Why is this function - targeting the VPC - specifically needed in the interface? and if it is, why not also have similar function for other resources relevant to the issue, say Subnets() and Nodes()? @zivnevo

Reply:
After discussing with @zivnevo we agreed to modify the interface so that `vpc()` returns `vpc` instead of name

Reply:
done